src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityFeed.test.js

```diff
@@ -148,6 +148,15 @@ describe('elements/content-sidebar/ActivityFeed/activity-feed/ActivityFeed', ()
         expect(wrapper.find('EmptyState').exists()).toBe(true);
     });
 
+    test('should not render empty state when UAA is enabled and there is 1 version (current version from file) and the start and end version are the same', () => {
```

Comment:
Was this meant to be "should render empty.." instead of "should not render"?

Reply:
Good catch